package.json

```diff
@@ -32,39 +32,39 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@headlessui/react": "^2.1.0",
-    "@heroicons/react": "^2.1.4",
-    "@netlify/plugin-nextjs": "^5.3.3",
-    "@next/third-parties": "^14.2.4",
-    "@storyblok/react": "^3.0.10",
+    "@headlessui/react": "^2.1.8",
+    "@heroicons/react": "^2.1.5",
+    "@netlify/plugin-nextjs": "^5.7.2",
+    "@next/third-parties": "^14.2.13",
+    "@storyblok/react": "^3.0.14",
     "@typeform/embed-react": "^3.17.0",
```

Comment:
This the other thing that I didn't update. Needs some troubleshooting for v4 so saving it for later.